trin-storage/src/sql.rs

```diff
@@ -0,0 +1,62 @@
+// SQLite Statements
+
+pub const CREATE_QUERY: &str = "CREATE TABLE IF NOT EXISTS content_data (
+                                content_id_long TEXT PRIMARY KEY,
+                                content_id_short INTEGER NOT NULL,
+                                content_key TEXT NOT NULL,
+                                content_value TEXT NOT NULL,
+                                network INTEGER NOT NULL DEFAULT 0,
+                                content_size INTEGER
+                            );
+                            CREATE INDEX content_size_idx ON content_data(content_size);
+                            CREATE INDEX content_id_short_idx ON content_data(content_id_short);
+                            CREATE INDEX content_id_long_idx ON content_data(content_id_long);
+                            CREATE INDEX network_idx ON content_data(network);";
+
+pub const INSERT_QUERY: &str =
+    "INSERT OR IGNORE INTO content_data (content_id_long, content_id_short, content_key, content_value, network, content_size)
+                            VALUES (?1, ?2, ?3, ?4, ?5, ?6)";
+
+pub const INSERT_LC_UPDATE_QUERY: &str =
+    "INSERT OR IGNORE INTO lc_update (period, value, score, update_size)
+                            VALUES (?1, ?2, ?3, ?4)";
+
+pub const DELETE_QUERY: &str = "DELETE FROM content_data
+                            WHERE content_id_long = (?1)";
+
+pub const XOR_FIND_FARTHEST_QUERY: &str = "SELECT
+                                    content_id_long
+                                    FROM content_data
+                                    WHERE network = (?2)
+                                    ORDER BY ((?1 | content_id_short) - (?1 & content_id_short)) DESC";
+
+pub const CONTENT_KEY_LOOKUP_QUERY: &str =
+    "SELECT content_key FROM content_data WHERE content_id_long = (?1) LIMIT 1";
+
+pub const CONTENT_VALUE_LOOKUP_QUERY: &str =
+    "SELECT content_value FROM content_data WHERE content_id_long = (?1) LIMIT 1";
+
+pub const TOTAL_DATA_SIZE_QUERY: &str = "SELECT TOTAL(content_size) FROM content_data";
+
+pub const TOTAL_ENTRY_COUNT_QUERY: &str =
+    "SELECT COUNT(content_id_long) FROM content_data WHERE network = (?1)";
+
+pub const PAGINATE_QUERY: &str =
+    "SELECT content_key FROM content_data ORDER BY content_key LIMIT :limit OFFSET :offset";
+
+pub const CONTENT_SIZE_LOOKUP_QUERY: &str =
+    "SELECT content_size FROM content_data WHERE content_id_long = (?1)";
+
+pub const CREATE_LC_UPDATE_TABLE: &str = "CREATE TABLE IF NOT EXISTS lc_update (
```

Comment:
*nitpick* Shouldn't this be `LC_UPDATE_CREATE_TABLE` to be more consistent with the other `lc_update` sql query names?